provider/resources.go

```diff
@@ -65,26 +65,27 @@ func Provider() tfbridge.ProviderInfo {
 		DisplayName:          "Materialize",
 		PreConfigureCallback: preConfigureCallback,
 		Config: map[string]*tfbridge.SchemaInfo{
-			"host": {
-				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_HOST"}},
-			},
-			"user": {
-				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_USER"}},
-			},
 			"password": {
 				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_PASSWORD"}},
 			},
-			"port": {
-				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_PORT"}},
-			},
 			"database": {
 				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_DATABASE"}},
 			},
+			"endpoint": {
+				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_ENDPOINT"}},
+			},
+			"cloud_endpoint": {
+				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_CLOUD_ENDPOINT"}},
+			},
+			"default_region": {
+				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_DEFAULT_REGION"}},
+			},
 			"sslmode": {
 				Default: &tfbridge.DefaultInfo{EnvVars: []string{"MZ_SSLMODE"}},
 			},
 		},
 		Resources: map[string]*tfbridge.ResourceInfo{
+			"materialize_app_password":                         {Tok: tfbridge.MakeResource(mainPkg, mainMod, "AppPassword")},
```

Comment:
Add any new Cloud resources to `Resources`. If there were datasources they would be added to `DataSources`.